workers/scanner.go

```diff
@@ -132,6 +136,7 @@ func (s *Scanner) threadedScanAndPin() {
 
 	// Main execution loop, goes on forever while the service is running.
 	for {
+		s.refreshMinPinners()
```

Comment:
This is not a requirement for this MR but something we should keep in the back of our heads is exploring mongo change streams (https://www.mongodb.com/docs/manual/changeStreams/#watch-a-collection--database--or-deployment).

If we ever have settings which we want to apply asap, that might be a good way to do it without polling.

Reply:
This looks nice. I'll put it in a F/U.